doc/setup.md

```diff
@@ -51,7 +51,19 @@ brew install geckodriver
 1. Install dependencies
 
 ```
-sudo apt install nginx supervisor postgresql libpq-dev nodejs
+sudo apt install nginx supervisor postgresql libpq-dev nodejs npm
+```
+
+If you do not have the python 3 developers tools: 
+
+```
+sudo apt install python3-dev
+```
+
+Also you will need to install astroplan:
+
+```
+pip install astroplan
```

Comment:
This should be installed automatically through the SkyPortal dependency checker.